web/views/index.php

```diff
@@ -7,28 +7,21 @@
     "color" => "brightgreen",
     "logo" => "youtube",
     "logoColor" => "white",
-    "style" => "flat-square",
+    "style" => "plastic",
     "format" => "short",
-    "label" => "YouTube view count",
+    "label" => "YouTube View Count",
     "labelColor" => "gray",
     "id" => "UCipSxT7a3rn81vGLw9lqRkg",
     "key" => "[YOUR API KEY HERE]" // you can put your API key here if you deploy it yourself
 ];
 
 // Query for finding view count in JSON
-$query = "$.items[0].statistics.viewCount";
+$query = "viewCount";
 
 // Build the Shields.io url using the above parameters and JSON query
 $url = getShieldURL($query, $defaults);
 
-// Set the content type for output
-header('Content-type: image/svg+xml');
```

Comment:
The header type needs to be set in order to be able to load the badge using markdown or HTML tags.

Reply:
Hey, @DenverCoder1, I have added these statements, please verify them.